Dockerfile

```diff
@@ -2,11 +2,12 @@ FROM python:3.9.12-slim-buster AS base
 WORKDIR /todo-app
 RUN pip3 install poetry
 COPY poetry.lock pyproject.toml wsgi.py ./
+RUN poetry config virtualenvs.create false --local
 
 FROM base AS production
 RUN poetry install --no-dev
 COPY todo_app ./todo_app
-ENTRYPOINT [ "poetry", "run", "gunicorn", "--bind", "0.0.0.0:80", "wsgi:start()" ]
+CMD poetry run gunicorn --bind 0.0.0.0:${PORT} "wsgi:start()" 
```

Comment:
While this works on Heroku, it will break your existing instructions if they don't specify a PORT variable! A nice fall back can be to specify the default port number as an environment variable, e.g. `ENV PORT=80` just above this line will provide a default of 80 but it can be overridden at run time if the user provides one